docs/hardware-consideration/Test-your-configuration.md

```diff
@@ -1,36 +1,29 @@
----
-custom_edit_url: null
----
-
-import Image from '/src/components/Images';
-import IconExternalLink from '@theme/IconExternalLink';
-
 # Test your hardware configuration
 
-To adapt Luos to your board specificities you need to configure it. This subject is covered in the [Luos configuration](/hardware-consideration/mcu.md).
+To adapt Luos to your board specificities, you need to configure it. This subject is covered in the [Luos configuration](/hardware-consideration/mcu.md).
 To validate your network hardware configuration, you can use a Luos tool called selftest available in the <a href="https://github.com/Luos-io/Luos" target="_blank">Luos Library<IconExternalLink width="10" /></a>.
 
 ## Hardware test conditions
 
-To allow selftest to check everything you need to have specific conditions for your board.
-To make this test you need to have only one board isolated (unconnected) from the network.
-Tx and Rx pin must be connected together or you have to be able to receive what you send. This is already the case in One Wire and RS485 configurations.
-Also you must connect PTP pins together.
+In order to allow selftest to check everything, you need to have specific conditions for your board.
+To make this test, you need to have only one board isolated (unconnected) from the network.
+Tx and Rx pins must be connected together, or you have to be able to receive what you send. This is already the case in One Wire and RS485 configurations.
+You must also connect the PTP pins together.
```

Comment:
the sentence after the word "or" is unclear: "be able to receive what you send". Did you mean "to receive on your own what you sent"? Please clarify

Reply:
Yes it's what I mean but I don't know how I can explain it differently